src/GeositeFramework/plugins/layer_selector_v2/plugin.json

```diff
@@ -0,0 +1,8 @@
+{
+    "css": [
+        "plugins/layer_selector_v2/main.css"
+    ],
+    "use": {
+        "underscore": { "attach": "_" }
```

Comment:
I don't know if it's possible here, but can we avoid use of `use`? In the future, it'll make it easier to transition to the built-in Dojo shimming system.